tests/ci/codebuild/release/al2_aarch64_test.yml

```diff
@@ -1,54 +0,0 @@
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-# SPDX-License-Identifier: Apache-2.0
-
-version: 0.2
-
-env:
-  variables:
```

Comment:
how is the removal of this and other codebuild specs related to removing dieharder tests?

Reply:
Not related. It was dead code, so I just removed those too. This is me being lazy and not creating two PRs. I can bring them back and submit another PR for it.

Reply:
nah, no worries.